about.html

```diff
@@ -0,0 +1,197 @@
+            <a href="#" class="top-bar__links"> Logout </a>
+          </li>
+        </ul>
+      </div>
+      <nav class="nav">
+        <div class="nav__container">
+          <a href="about.html" class="logo">
+            <img
+              src="./images/africalogo.jpg"
+              alt="Africa logo"
+              class="logo-img"
+            />
+          </a>
+
+          <div class="nav__mobile">
+            <input type="checkbox" id="check" class="nav__mobile--check" />
+            <label for="check" class="nav__mobile--label">
+              <span class="nav__mobile--icon">&nbsp;</span>
+            </label>
+
+            <div class="nav__mobile--sidebar">
+              <div class="nav__mobile--header">
+                <span>Quick Links</span>
+              </div>
+              <ul class="nav__mobile--menu">
+                <li class="nav__mobile--list">
+                  <a href="about.html#aboutto" class="nav__mobile--links">About</a>
+                </li>
+                <li class="nav__mobile--list">
+                  <a href="index.html#program" class="nav__mobile--links">Program</a>
+                </li>
+                <li class="nav__mobile--list">
+                    <a href="#" class="nav__mobile--links">Join</a>
+                  </li>
+                <li class="nav__mobile--list">
+                  <a href="index.html#sponsors" class="nav__mobile--links">Sponsor</a>
+                </li>
+                <li class="nav__mobile--list">
+                  <a href="index.html#speakers" class="nav__mobile--links">Speakers</a>
+                </li>
+                <li class="nav__mobile--list">
+                  <a href="about.html#check" class="nav__mobile--links">Africa Rise</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+
+          <ul class="nav__menu">
+            <li class="nav__list">
+              <a href="about.html#aboutto" class="nav__links">About</a>
+            </li>
+            <li class="nav__list">
+              <a href="#program" class="nav__links">Program</a>
+            </li>
+            <li class="nav__list">
+                <a href="#" class="nav__links">Join</a>
+              </li>
+            <li class="nav__list">
+              <a href="index.html#sponsors" class="nav__links">Sponsor</a>
+            </li>
+            <li class="nav__list">
+              <a href="#speakers" class="nav__links">Speakers</a>
+            </li>
+            <li class="nav__list">
+              <a href="about.html#check" class="nav__links">Africa Rise</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="intro">
+      <div class="header__content">
+        <ul class="hoo">
+          <li><img src="images/home.png" alt="home" class="home"></li>
+          <li class="dash">&gt;</li>
+          <li>조선말</li>
+          <li class="dash">&gt;</li>
+          <li>CC조선말</li>
+      </ul>
```

Comment:
- [ ] Nice work trying to implement a breadcrumb component, but please make sure that you understand what you are doing, is not good to mix together Korean language on an English site.

Reply:
Hello @williamrolando88, concerning this i was following the design not doing things i don't understand